### PR TITLE
Contribute section

### PR DESCRIPTION
**Pull request by symeneses**
It improves the contribute section.

### CODE REVIEW

File: contributing/index.md [hunk 1]
```diff
@@ -1,45 +1,74 @@
...
-% TODO: make nicer and add missing pages/sections
+:::{toctree}
+:maxdepth: 1
+:hidden:
 
-## ...with the website
-To build the website locally, install the requirements
+self
+../about/contributing_to_pymc/index
+../about/contributing_to_documentation/docs_presentation
```
Comment (OriolAbril) [inline contributing/index.md:10]: I would link to the index page instead so readers can choose to read, watch the video or both.

File: contributing/index.md [hunk 2]
```diff
@@ -1,45 +1,74 @@
...
+
+### Some outreach related advise:
 
-Some outreach related advise:
 * Always use [https://pymc-data-umbrella.xyz](https://pymc-data-umbrella.xyz) when linking
   to the general series of events, use links within this domain for specific links to single
   events or external links (i.e. direct link to webinar or to registration form).
 
   **DO NOT USE** `pymc-data-umbrella.readthedocs.io` nor `sprint-pymc-data-umbrella.readhedocs.io`
   domains.
```
Comment (reshamas) [inline contributing/index.md:57]:
```suggestion
```
@OriolAbril 
Can we remove this? I think it's more distracting that useful.  It is residual from the very initial website, which is not relevant anymore.

File: contributing/index.md [hunk 3]
```diff
@@ -1,45 +1,74 @@
...
   If you need some alternative to "webinar" consider using "event" or "workshop" before for example.
 
-## ...paving the way of new contributors
-Help with issue triaging on both pymc and pymc-examples repo
+## Paving the way of new contributors
+
+Help with issue triaging on both [pymc](https://github.com/pymc-devs/pymc) and [pymc-examples](https://github.com/pymc-devs/pymc-examples) repo.
 
-## ...translating content
+## Translating content
```
Comment (reshamas) [inline contributing/index.md:68]:
```suggestion
## Translating content (set-up in progress)
```

File: contributing/index.md [hunk 4]
```diff
@@ -1,45 +1,74 @@
...
 
 :::{note}
-We first need to set up the translation infrastructure with sphinx, which we have never
-done before. If you know how to do it, please reach out.
+We first need to set up the translation infrastructure with sphinx. If you know how to do it or are interested in translating, please reach out.
 
 If you'd like to volunteer with translations only, let us know too, but we ask you to be
 patient for now. There is nothing yet for translators to work on.
```
Comment (reshamas) [inline contributing/index.md:74]:
```suggestion
In the meantime, if you would to volunteer for translations, please add your information to [Issue #99](https://github.com/pymc-devs/pymc-data-umbrella/issues/99) on GitHub. We will be in touch when we are ready to move forward on this project.
```

File: contributing/index.md [hunk 5]
```diff
@@ -1,45 +1,74 @@
...
   sprint session.
 * Use "webinars" to refer to the webinars we are hosting in preparation of the sprint. Try
   to avoid "tutorials" as it can be confusing with the content of the documentation webinar.
   If you need some alternative to "webinar" consider using "event" or "workshop" before for example.
 
-## ...paving the way of new contributors
-Help with issue triaging on both pymc and pymc-examples repo
+## Paving the way of new contributors
+
+Help with issue triaging on both [pymc](https://github.com/pymc-devs/pymc) and [pymc-examples](https://github.com/pymc-devs/pymc-examples) repo.
```
Comment (reshamas) [inline contributing/index.md:66]: @OriolAbril 
Are there any triaging instructions or resources available to share?

File: contributing/index.md [hunk 6]
```diff
@@ -1,45 +1,74 @@
...
+
+## Spread the word
+
+Share the announcements on social networks, within your personal networks. This will help us to reach more people.
+
+<a href="https://twitter.com/intent/tweet?text=https://pymc-data-umbrella.xyz">
+  Create Tweet <i class="fab fa-twitter h4 text-main"></i>
+</a>
+
+### Some outreach related advise:
```
Comment (reshamas) [inline contributing/index.md:50]:
```suggestion
### Some outreach related advise
```

**Comment by reshamas** [comment]
@OriolAbril @symeneses Overall comment here on text for links.  

#### Short story
Do *not* include links in this format:
`[here][https://xxx.xxx.xxx]`

Do provide links with descriptive text:
```
To support Data Umbrella and keep it running, please donate to the [Data Umbrella Collective](https://opencollective.com/data-umbrella)
```

#### Longer story
>Accessible Hyperlinks - What are They? Who Benefits?
>When there are links in a document, screen reader users can pull up a list of links embedded in the document
>Descriptive text provides context for all users and people who are: Blind/Low vision

Reference:  https://github.com/scikit-learn/scikit-learn/issues/21214#issuecomment-949678477

File: contributing/index.md (hunk 1 above)
Comment (symeneses) [reply contributing/index.md]: Done!

File: contributing/index.md [hunk 7]
```diff
@@ -1,45 +1,74 @@
...
+You can use as reference the {ref}`PR Tutorial <pr_tutorial>` which explains how you can do a PR to PyMC.
+:::
+
+## Spread the word
+
+Share the announcements on social networks, within your personal networks. This will help us to reach more people.
+
+<a href="https://twitter.com/intent/tweet?text=https://pymc-data-umbrella.xyz">
+  Create Tweet <i class="fab fa-twitter h4 text-main"></i>
+</a>
```
[the remark this answers is not in the document]
Comment (symeneses) [reply contributing/index.md]: Both suggestions applied!

File: contributing/donate.md [hunk 8]
```diff
@@ -0,0 +1,18 @@
+# Support us
```
[the remark this answers is not in the document]
Comment (symeneses) [reply contributing/donate.md:1]: I have also used star 😆. Checking...

File: contributing/index.md (hunk 2 above)
Comment (OriolAbril) [reply contributing/index.md:59]: yes, we can remove this

File: contributing/index.md (hunk 5 above)
Comment (OriolAbril) [reply contributing/index.md:68]: No, and only recurrent contributors and people on the pymc team can triage issues.

File: contributing/index.md [hunk 9]
```diff
@@ -1,46 +1,76 @@
...
   If you need some alternative to "webinar" consider using "event" or "workshop" before for example.
 
-## ...paving the way of new contributors
-Help with issue triaging on both pymc and pymc-examples repo
+## Paving the way of new contributors
+
+Help with issue triaging on both [pymc](https://github.com/pymc-devs/pymc) and [pymc-examples](https://github.com/pymc-devs/pymc-examples) repo.
 
-## ...translating content
+## Translating content (set-up in progress)
```
Comment (OriolAbril) [inline contributing/index.md:70]: I'd maybe use "waiting for volunteers" or "volunteers needed". As things are now, I see it more as an issue of more people being needed to translate than an issue of time.

File: contributing/donate.md (hunk 8 above)
[the remark this answers is not in the document]
Comment (symeneses) [reply contributing/donate.md:1]: Done! For some reason `octicon` works but not any `material-`.

File: contributing/index.md (hunk 2 above)
Comment (symeneses) [reply contributing/index.md]: Done!

File: contributing/donate.md (hunk 8 above)
[the remark this answers is not in the document]
Comment (symeneses) [reply contributing/donate.md:1]: Locally, I updated `sphinx-design` to 0.2.0 and it works now.

File: contributing/index.md (hunk 9 above)
Comment (symeneses) [reply contributing/index.md]: I think it's better, it's more aligned with the text we have there now. Changed.

File: contributing/index.md [hunk 10]
```diff
@@ -1,46 +1,76 @@
...
+
+## Spread the word
+
+Share the announcements on social networks, within your personal networks. This will help us to reach more people.
+
+:::{div} sd-text-center sd-fs-3
+[{fab}`twitter`](https://twitter.com/intent/tweet?text=https://pymc-data-umbrella.xyz)
+:::
+
+There are social media links and text available in the {ref}`Community Partner section <become_community_partner>` which can be shared as well.
```
[the remark this answers is not in the document]
Comment (symeneses) [reply contributing/index.md:56]: We can do this or find another solution in a different PR to solve #101

**Pull request merged**